README.md

```diff
@@ -95,6 +95,8 @@ The full list is below.
 | `linode_nodebalancer_transfer_total_bytes`   | Gauge   |
 | `linode_nodebalancer_transfer_out_bytes`     | Gauge   |
 | `linode_nodebalancer_transfer_in_bytes`      | Gauge   |
+| `linode_objectstorage_objects_count`         | Gauge   |
+| `linode_objectstorage_size`                  | Gauge   |
```

Comment:
Should be `linode_objectstore_size_bytes`.

This is a good reminder: I should review the existing metrics and add [base units](https://prometheus.io/docs/practices/naming/#base-units) to metrics where applicable.

Reply:
Yes and `linode_instance_disk` (MB).

However, I'm unsure what the correct values should be because Linode uses GB and MB seemingly incorrectly.

It reports that a 25 GB disk as 25600 MB rather than 25000 MB.

And `fdisk`'ing the same disk (correctly) reports (!) 24.5 GiB, 26306674688 bytes

Which should be either 26843545600 bytes if it's 25 GiB

Or 25000000000 bytes if it's 25 GB